app/src/main/java/nerd/tuxmobil/fahrplan/congress/repositories/AppRepository.kt

```diff
@@ -91,22 +104,22 @@ object AppRepository {
 
     fun loadSchedule(url: String,
                      okHttpClient: OkHttpClient,
-                     onFetchingDone: (fetchScheduleResult: FetchScheduleResult) -> Unit,
+                     onFetchingDone: (httpStatus: HttpStatus) -> Unit,
                      onParsingDone: (parseScheduleResult: ParseResult) -> Unit,
                      onLoadingShiftsDone: (loadShiftsResult: LoadShiftsResult) -> Unit
     ) {
-        check(onFetchingDone != {}) { "Nobody registered to receive FetchScheduleResult." }
+        check(onFetchingDone != {}) { "Nobody registered to receive HttpStatus." }
```

Comment:
I don't think this condition will ever evaluate to false.

```kotlin
@Test
fun lambdaEquals() {
    Assert.assertEquals({}, {})
}
```

```
java.lang.AssertionError: expected: nerd.tuxmobil.fahrplan.congress.QuickTest$lambdaEquals$1<Function0<kotlin.Unit>> but was: nerd.tuxmobil.fahrplan.congress.QuickTest$lambdaEquals$2<Function0<kotlin.Unit>>
Expected :nerd.tuxmobil.fahrplan.congress.QuickTest$lambdaEquals$1<Function0<kotlin.Unit>> 
Actual   :nerd.tuxmobil.fahrplan.congress.QuickTest$lambdaEquals$2<Function0<kotlin.Unit>>
```

Well, it might if R8 merges the two lambda classes. But that's not something you want to depend on.